### PR TITLE
CIP-1854 | Adjust preamble and structure w.r.t CIP-0001

### LINKED ISSUE #726: CIP-1854 | Adjust preamble and structure w.r.t CIP-0001
Fixed by #727.

### PR DESCRIPTION
**Pull request by rphair**
Fixes #726.

([updated proposal](https://github.com/rphair/CIPs/blob/remediate-1854/CIP-1854/README.md))

### CODE REVIEW

**Comment by rphair** [comment]
@siegfried as per https://github.com/cardano-foundation/CIPs/pull/617#issuecomment-1868557630 I'm going to try cutting through the bureaucracy and reclassifying this as `Active` due to implementation in ADAO / Round Table wallet.  

The fact that it's open source also qualifies it as a reference implementation in my opinion, although the CIP-1854 authors were not able to provide an official reference implementation.

Other editors & reviewers: I think having this CIP on the books as `Active`, especially after the flurry of social media postings 7 months ago declaring Cardano multisig and/or CIP-1854 to be unimplemented, will be healthy for the community and will encourage further development in the multisig wallet space.

**Review by Ryun1 — APPROVED** [review]
LGTM

File: CIP-1854/README.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
-Authors: Matthias Benkort <matthias.benkort@iohk.io>, Pawel Jakubas <pawel.jakubas@iohk.io>
-Comments-Summary: Multi-party transaction signing and key management for HD wallets.
-Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-1854
-Status: Draft
-Type: Standards
+Status: Active
+Category: Wallets
+Authors:
+  - Matthias Benkort <matthias.benkort@iohk.io>
+  - Pawel Jakubas <pawel.jakubas@iohk.io>
```
Comment (KtorZ) [inline CIP-1854/README.md:8]:
```suggestion
  - Matthias Benkort <matthias.benkort@cardanofoundation.org>
  - Pawel Jakubas <pawel.jakubas@cardanofoundation.org>
```

File: CIP-1854/README.md [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
-Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-1854
-Status: Draft
-Type: Standards
+Status: Active
+Category: Wallets
+Authors:
+  - Matthias Benkort <matthias.benkort@cardanofoundation.org>
+  - Pawel Jakubas <pawel.jakubas@cardanofoundation.org>
+Implementors:
+  - Round Table wallet
```
Comment (KtorZ) [inline CIP-1854/README.md:10]:
```suggestion
  - Round Table wallet
  - Daedalus 
  - Yoroi
```

File: CIP-1854/README.md [hunk 3]
```diff
@@ -1,31 +1,34 @@
...
+  - Daedalus 
+  - Yoroi
```
Comment (KtorZ) [inline CIP-1854/README.md:12]:
```suggestion
```

**Review by KtorZ — APPROVED** [review]

**Pull request merged**